db/connect.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from "mongoose";
+
+const MONGODB_URI = process.env.MONGODB_URI;
+if (!MONGODB_URI) {
+  console.log(MONGODB_URI, "WTF COUNT?");
```

Comment:
You should remove unnecessary logging.
suggestion```

```

Reply:
Thanks, removed it!